static/src/js/schemas/otherSchemasVisSchema.js

```diff
@@ -0,0 +1,2479 @@
+/* eslint-disable no-template-curly-in-string */
+const otherSchemasVisSchema = {
```

Comment:
I went through this schema and compared it to the remote ones the usage of `oneOf` makes sense to me I think this is correct similarly the values which were in the https://git.earthdata.nasa.gov/projects/EMFD/repos/otherschemas/browse/visualization/v1.1.0/definitions.json looked correct on here